src/OOVPADatabase/Xapi/4831.inl

```diff
@@ -206,6 +206,32 @@ OOVPA_SIG_MATCH(
     //
 );
 
+// ******************************************************************
+// * XInputSetLightgunCalibration
+// ******************************************************************
+OOVPA_SIG_HEADER_XREF(XInputSetLightgunCalibration,
+                      4831,
+
+                      XRefNoSaveIndex,
```

Comment:
With full xreference cache system pull request merged, these lines should be removed.
```suggestion
```